ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml

```diff
@@ -41,7 +41,7 @@ tests:
     workflow: openshift-qe-installer-aws
   timeout: 7h0m0s
 - as: control-plane-24nodes
-  cron: 0 7 2 * *
+  cron: 0 8 9 * *
```

Comment:
Would you mind leaving hour=7 ? This will make 5 jobs at hour=8 and while they are all on different days, it is really not necessary.

Reply:
Done.